.changeset/smart-buttons-applaud.md

```diff
@@ -0,0 +1,5 @@
+---
+"@salt-ds/lab": patch
+---
+
+Update Slider component implementation to have separate Slider and Range Slider components, with improved accessibility and test coverage.
```

Comment:
If possible,  add a diff of before / after to help migrate

Reply:
"Range Slider" to `RangeSlider` so we create familiarity with the component names and wrap `Slider` in backticks, so both stand out as component names.